sql/moz-fx-data-shared-prod/firefox_desktop_derived/clients_city_seen_v1/query.sql

```diff
@@ -0,0 +1,144 @@
+ -- Query generated via sql_generators.clients_city_seen.
+ -- this mimics the logic used in baseline_clients_daily_v1.
+ -- some client_id do not have first_seen_geo_date since stable tables only have 2 years of data.
+{% if is_init() %}
+  WITH base_firefox_desktop AS (
+    SELECT
+      submission_timestamp,
+      DATE(submission_timestamp) AS submission_date,
+      LOWER(client_info.client_id) AS client_id,
+      sample_id,
+      mozfun.glean.parse_datetime(ping_info.end_time) AS parsed_end_time,
+      `moz-fx-data-shared-prod.udf.glean_timespan_seconds`(
+        metrics.timespan.glean_baseline_duration
+      ) AS duration,
+      metadata.geo.city,
+      metadata.geo.subdivision1 AS geo_subdivision1,
+      metadata.geo.subdivision2 AS geo_subdivision2,
+    FROM
+      `moz-fx-data-shared-prod.firefox_desktop_stable.baseline_v1`
+    WHERE
+      sample_id = 0
+  ),
+  with_dates_firefox_desktop AS (
+    SELECT
+      *,
+      DATE(SAFE.TIMESTAMP_SUB(parsed_end_time, INTERVAL duration SECOND)) AS session_start_date,
+      DATE(parsed_end_time) AS session_end_date,
+    FROM
+      base_firefox_desktop
+  ),
+  with_date_offsets_firefox_desktop AS (
+    SELECT
+      *,
+      DATE_DIFF(submission_date, session_start_date, DAY) AS session_start_date_offset,
+      DATE_DIFF(submission_date, session_end_date, DAY) AS session_end_date_offset,
+    FROM
+      with_dates_firefox_desktop
+  ),
+  overactive_firefox_desktop AS (
+    SELECT
+      submission_date,
+      client_id
+    FROM
+      with_date_offsets_firefox_desktop
+    WHERE
+      submission_date >= "2025-08-14"
+    GROUP BY
+      submission_date,
+      client_id
+    HAVING
+      COUNT(*) > 150000
+  ),
+  windowed_firefox_desktop AS (
+    SELECT
+      submission_date,
+      client_id,
+      sample_id,
```

Comment:
`sample_id` seems redundant as it's always the same per `client_id`. You can consider removing it for lower costs (hashing, comparing)